env update

alfrediii opened the pull request.

alfrediii committed:
env update

alfrediii committed:
Update schema-patient.sql

tag commented:
Your merge request is backward. You probably want to merge from my repo to yours. Instead, this request attempts to merge your content into my repo.

Also, why did you delete `env.example`? The file poses no harm by being there, and it's intended to exist for documentation reasons.

alfrediii commented:
> Your merge request is backward. You probably want to merge from my repo to yours. Instead, this request attempts to merge your content into my repo.
> 
> Also, why did you delete `env.example`? The file poses no harm by being there, and it's intended to exist for documentation reasons.

Sorry! I don't remember why I did that. What you're saying makes sense. 

I deleted it because I didn't see a use for it after I updated my environment file. Please forgive me.
